### PR TITLE
scopeWhereTranslationLike

### PR DESCRIPTION
**Pull request by sabau**
Hi Dismav, 
I'm currently using your package in some of my projects.
In order to implement easily some search functions I currently use this scope: WhereTranslationLike
I hope it can be useful to someone else :)

Bye,
Karoly

### CODE REVIEW

File: tests/ScopesTest.php [hunk 1]
```diff
@@ -91,4 +91,23 @@ public function test_whereTranslation_filters_by_translation_and_locale()
...
+        /** @var Country $country */
+        $country = Country::whereTranslationLike('name', '%Greec%')->first();
+        $this->assertSame('gr', $country->code);
+    }
+
+    public function test_whereTranslationLike_filters_by_translation_and_locale()
+    {
+        Country::create(['code' => 'some-code', 'name' => 'Griechenland']);
+
+        /** @var Country $country */
```
Comment (dimsav) [inline tests/ScopesTest.php]: Copy paste? :P

**Comment by dimsav** [comment]
Excellent! Fix the dummy docblock and ready to :ship:

File: tests/ScopesTest.php (hunk 1 above)
Comment (sabau) [inline tests/ScopesTest.php]: Yep :) just an extra scope

**Comment by sabau** [comment]
Sorry, I forgot to update the comment.

File: tests/ScopesTest.php (hunk 1 above)
Comment (dimsav) [inline tests/ScopesTest.php]: I meant this line:

```
/** @var Country $country */
```

**Comment by dimsav** [comment]
Awesome, thanks so much!

**Pull request merged**